src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -0,0 +1,313 @@
+        color: Colors.black[50],
+        borderColor: Colors.black[50],
+        [theme.breakpoints.up('sm')]: {
+            width: '80px',
+        },
+    },
+    formControl: {
+        width: '100%',
+    },
+    radioLabel: {
+        display: 'flex',
+    },
+    icon: {
+        fill: Colors.black[200],
+        marginLeft: `${theme.spacing(0.5)}px`,
+    },
+    iconContainer: {
+        marginRight: `${theme.spacing(2)}px`,
+        maxWidth: '40px',
+        minWidth: '40px',
+        width: '40px',
+        marginTop: `${theme.spacing(1)}px`,
+    },
+    formLabel: {
+        margin: 0,
+        width: '100%',
+        padding: theme.spacing(1),
+        borderBottom: `1px solid ${theme.palette.divider}`,
+        '&:last-child': {
+            borderBottom: 'none',
+        },
+    },
+    toolbarGutters: {
+        padding: `0 ${theme.spacing(2)}px`,
+    },
+    placementOfList: {
+        position: 'absolute',
+        bottom: theme.spacing(3),
+        right: theme.spacing(3),
+        [theme.breakpoints.down('xs')]: {
+            bottom: 0,
+            right: 0,
+            width: '100%',
+        },
+    },
+    infoList: {
+        marginBottom: theme.spacing(2),
+        '&:last-child': {
+            marginBottom: 0,
+        },
+    },
+    subTitle: {
+        color: Colors.black[200],
+    },
+    snackbar: {
+        position: 'inherit',
+        transform: 'none',
+    },
+    SnackbarContent: {
+        borderRadius: 0,
+        padding: `0 ${theme.spacing(2)}px 0 ${theme.spacing(1)}px`,
+    },
+    messageContainer: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+    messageTextContainer: {
+        marginRight: `${theme.spacing(3)}px`,
+    },
+}));
+const createFileItem = (increment: number): FolderItem => ({
+    id: increment,
+    open: true,
+    duration: null,
+    buttonLabel: 'Close',
+    name: 'PX Blue is Awesome.pdf',
+    progress: 0,
+    status: `Uploading (0%)`,
+});
+
+export const ProgressBar = (): JSX.Element => {
+    const dispatch = useDispatch();
+    const classes = useStyles();
+    const [fileUploadList, setFileUploadList] = useState<FolderItem[]>(uploadFileList);
+
+    const [radioButtonvalue, setRadioButtonvalue] = useState('1');
+    const [count, setCount] = useState(0);
+    const changeRadioGroup = (event: React.ChangeEvent<HTMLInputElement>): void => {
+        setRadioButtonvalue((event.target as HTMLInputElement).value);
+    };
+
+    const uploadFile = useCallback((): void => {
+        setCount(count + 1);
+        setFileUploadList((oldList) => [...oldList, createFileItem(count)]);
+    }, [fileUploadList, setFileUploadList]);
+
+    const removInfoList = useCallback(
+        (index: number, status: string): void => {
+            if (status === 'Complete') {
+                return;
+            }
+            setFileUploadList((oldList) => oldList.filter((item) => item.id !== index));
+        },
+        [fileUploadList]
```

Comment:
You don't use fileUploadList anywhere, so it's not necessary in the dependencies array
```suggestion
        []
```